enterprise/internal/licensing/data.go

```diff
@@ -40,9 +40,13 @@ const (
 	// FeatureBranding is whether custom branding of this Sourcegraph instance has been purchased.
 	FeatureBranding Feature = "branding"
 
-	// FeatureCampaigns is whether campaigns on this Sourcegraph instance has been purchased.
+	// DEPRECATED: See FeatureBatchChanges.
+	// FeatureCampaigns is whether campaigns (now: batch changes) on this Sourcegraph instance has been purchased.
```

Comment:
```suggestion
	// FeatureCampaigns is whether campaigns (now: batch changes) on this Sourcegraph instance has been purchased.
	//
	// DEPRECATED: See FeatureBatchChanges.
```

If you order lines this way, some IDE (e.g. GoLand) would visually strikethrough the word to identify its deprecation.

Reply:
Cool! I was wondering about that.